src/pages/home/report/EmojiSkinToneList.js

```diff
@@ -25,28 +22,31 @@ const propTypes = {
         fontSize: PropTypes.number,
     }).isRequired,
 
-    toggleSkinToneList: PropTypes.func.isRequired,
-
-
     /** Props related to translation */
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
-    isSkinToneListVisible: false,
 };
 
 const EmojiSkinToneList = (props) => {
     const selectedEmoji = getSkinToneEmojiFromIndex(props.preferredSkinTone);
     const [highlightedIndex, setHighlightedIndex] = useState(selectedEmoji.skinTone);
```

Comment:
We don't use hooks like `useState` in our codebase, so this component will have to be changed. Sorry for not noticing it in my earlier review 😞

Reply:
Ohh okay. I'll change it to a class component. Any other changes?

Reply:
Not that I can see at the moment. Thanks!